configuration/configuration.go

```diff
@@ -270,11 +254,44 @@ type DataConfiguration struct {
 	// at the examples directory for an example of how to structure this file.
 	// default: ""
 	InterestingAccounts string `json:"interesting_accounts"`
+
+	// ReconciliationDisabled is a boolean that indicates reconciliation should not
+	// be attempted. When first testing an implementation, it can be useful to disable
+	// some of the more advanced checks to confirm syncing is working as expected.
+	ReconciliationDisabled bool `json:"reconciliation_disabled"`
```

Comment:
thoughts on renaming to Affirmative instead of Negative
```suggestion
	ReconciliationEnabled bool `json:"reconciliation_enabled"`
```

Reply:
This is so the default parsing is `false`...if we didn't have this, people would have to stub a ton of defaults in their config file